imports/plugins/core/payments/server/no-meteor/schemas/schema.graphql

```diff
@@ -68,6 +71,9 @@ type Payment implements Node {
   "The payment method"
   method: PaymentMethod!
 
+  "Risk level of payment"
+  riskLevel: String
```

Comment:
It looks like there are three possible values mentioned in the code: normal, elevated, highest. Maybe make this an enum value so clients know what to expect back here?

Reply:
Updated